core/tests/protractor_utils/general.js

```diff
@@ -72,7 +72,10 @@ var checkForConsoleErrors = function(errorsToIgnore) {
 var isInDevMode = function() {
   browser.get('/splash');
   waitFor.pageToFullyLoad();
-  return browser.executeScript('return constants.DEV_MODE');
+  return browser.executeScript(
```

Comment:
This looks a bit complex. Some thoughts:

Can we put a CSS class on the dev mode indicator and just check for that element (like in normal e2e tests) -- if the intention is to detect the current page status?

On the other hand, judging by the previous code, it might be that this is to check how the test was configured. If that's the case, we do that configuration by passing an arg to the e2e test script, right? Could we just set things up so that we can read that configuration from here?